src/core/socket.c

```diff
@@ -1646,7 +1646,7 @@ static int socket_open_fds(Socket *orig_s) {
 
                         case SOCK_STREAM:
                         case SOCK_SEQPACKET:
-                                if (s->socket_protocol == IPPROTO_SCTP)
+                                if (IN_SET(s->socket_protocol, IPPROTO_SCTP, IPPROTO_MPTCP))
                                         p->address.protocol = s->socket_protocol;
                                 break;
```

Comment:
Note that `IPPROTO_MPTCP` does not support `SOCK_SEQPACKET`.

Reply:
Good catch, fixed!